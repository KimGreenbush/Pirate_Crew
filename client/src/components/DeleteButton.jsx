import React from "react"
import axios from "axios"

const DeleteButton = ({pirateId, successCallback}) => {
    const deletePirate = e => {
        axios.delete(`http://localhost:9001/api/pirates/${pirateId}`)
            .then(res => successCallback())
    }

    return (
        <button onClick={deletePirate}>Walk the Plank</button>
    )
}

export default DeleteButton