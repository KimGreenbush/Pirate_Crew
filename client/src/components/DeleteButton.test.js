/**
 * @jest-environment jsdom
 */

import React from "react";
import DeleteButton from "./DeleteButton";
import renderer from "react-test-renderer";

test("renders DeleteButton without crashing", () => {
	const component = renderer.create(<DeleteButton />);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
