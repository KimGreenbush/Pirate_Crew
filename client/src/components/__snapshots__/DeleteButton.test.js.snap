// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders DeleteButton without crashing 1`] = `
<button
  onClick={[Function]}
>
  Walk the Plank
</button>
`;
